Package.swift

```diff
@@ -49,19 +51,23 @@ let package = Package(
             name: "SpeziLLMLocal",
             dependencies: [
                 .target(name: "SpeziLLM"),
-                .product(name: "llama", package: "llama.cpp"),
                 .product(name: "SpeziFoundation", package: "SpeziFoundation"),
-                .product(name: "Spezi", package: "Spezi")
-            ],
-            swiftSettings: [
-                .interoperabilityMode(.Cxx)
```

Comment:
We might want to adjust the README to remove the C++ interoperability references and elements that are currently required. We should generally update the README with this PR and rework it bit.

Reply:
I've started on removing all C++/Llama references from the README. Maybe we want to include a short section and introduce MLX. I will update that in the next commits.